libcore/src/HistoryParser.cpp

```diff
@@ -21,6 +21,11 @@
 #include "HistoryParser.h"
 #include "FileUtils.h"
 
+const QString HistoryParser::historyFile = "/history.json";
+const QString HistoryParser::speedProperty = "speed";
```

Comment:
This could be also done with the `AddonItemModel` class (in `app/src/AddonItemModel.cpp`) which reads the JSON file of a downloaded addon, and the `AddonManager` class (in `app/src/AddonManager.cpp`) which reads the JSON files of installed addons.

Reply:
`AddonItemModel` - done

Reply:
`AddonManager` - done